### PR TITLE
docs: added "commerical" to API readme

### PR DESCRIPTION
**Pull request by PeerRich**

### CODE REVIEW

**Review by pumfleet — DISMISSED** [review]
Wrong spelling

**Review by alannnc — DISMISSED** [review]

**Comment by alannnc** [comment]
> Wrong spelling

I think from a legal point the word is ok, right?

File: apps/api/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <a href="https://cal.com/docs/enterprise-features/api#api-server-specifications">Read the API docs</a>
 </div>
 
-# Cal.com Public API 
+# Commerical Cal.com Public API
```
Comment (PeerRich) [inline apps/api/README.md:10]:
```suggestion
# CommercialCal.com Public API 
```

File: apps/api/README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
   <a href="https://cal.com/docs/enterprise-features/api#api-server-specifications">Read the API docs</a>
 </div>
 
-# Cal.com Public API 
+# CommercialCal.com Public API
```
Comment (PeerRich) [inline apps/api/README.md:10]:
```suggestion
# Commercial Cal.com Public API 
```

**Review by sean-brydon — APPROVED** [review]

**Comment by PeerRich** [comment]
i am kinda amazed by me how i had one task, to add a single word, and failed hahahah

**Pull request merged**